### DISCUSSION TITLE
DBeaver for Microsoft store has no menu to install Office plugin

### DISCUSSION THREAD

**Discussion by RichardSCarchitect** [dbeaver/dbeaver#21158] [Ideas]
### Description

Used DBeaver from zip file in Windows 10 professional for a few years.  Today I tried to install from Microsoft store, everything works except there is no way to install office plugin eg. export to Excel.  I tried to run DBeaver as administrator - no dice.  Is there a workaround to install Office plugin?  Thanks

### DBeaver Version

23.1.4

### Operating System

Windows 10 professional

### Database and driver

SAP HANA, Databricks

### Steps to reproduce

Download DBeaver from Windows store, run, open database and try to export a table content to Excel

**Comment by LonwoLonwo** [comment]
Hello @RichardSCarchitect 

Unfortunately, there is no workaround to do it. This is the politic of Windows Store - you can not modify the downloaded application. It is read-only.
The best way to use the Office plugin - is to choose any other DBeaver installations that support extra plugins.

**Comment by RichardSCarchitect** [comment]
Thanks very much for explanation!
